tests/roots/test-ext-autodoc/index.rst

```diff
@@ -11,3 +11,16 @@
 .. autofunction:: target.typehints.incr
 
 .. autofunction:: target.typehints.tuple_args
+
+.. autodata:: target.typed_vars_py35.attr1
```

Comment:
Basically, we don't need to update this file because we have `do_autodoc()` helper.